man/systemd-repart.xml

```diff
@@ -372,11 +372,17 @@
         are excluded. If <option>--exclude-partitions=</option> is used, all partitions that are specified
         are excluded. Both options take a comma separated list of GPT partition type UUIDs or identifiers
         (see <varname>Type=</varname> in
-        <citerefentry><refentrytitle>repart.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>). All
-        partitions that are excluded using these options are still taken into account when calculating the
-        sizes and offsets of other partitions, but aren't actually written to the disk image. The net effect
-        of this option is that if you run <command>systemd-repart</command> again without these options, the
-        missing partitions will be added as if they had not been excluded the first time
+        <citerefentry><refentrytitle>repart.d</refentrytitle><manvolnum>5</manvolnum></citerefentry>).
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--skip-partitions=</option><arg rep="repeat">PARTITION</arg></term>
```

Comment:
can we find a more descriptive word? maybe "--ghost-partitions=" or "--void-partitions="? "--reserve-partitions="?

Reply:
--phantom-partitions?

Reply:
shadow? vacate?

Reply:
--blank-partitions=? (I think so far the best, dunno?)

Reply:
Frankly I still think --skip-partitions sounds better than all of the above.

Reply:
I think skip works best too, it conveys exactly what it does

Reply:
it doesn't. because the partitions are not skipped after all, they are taken into full consideration with only a single thing different: the partitions are not actually written out.

Ask a random hacker what he'd guess what the difference between "--exclude-partition=" and "--skip-partition=" might be. I doubt anyone would guess that.

I think "--ghost-partition=" or "--blank-partition=" could convey the difference much better: it's clear that the partition *is* considered, except not realized in the end.